icons/tauri/tauri-plain-wordmark.svg

```diff
@@ -0,0 +1 @@
4.367-.562s.43-.227.73-.227h4.152c.301 0 .543.074.73.227a1.04 1.04 0 0 1 .367.563l6.57 18.027.055.25c0 .152-.062.293-.195.422a.57.57 0 0 1-.422.168h-3.453c-.449 0-.75-.195-.898-.59L75.151 71.9h-7.5l-1.066 2.863c-.148.395-.457.59-.926.59zm11.82-7.496l-2.613-7.414-2.609 7.414zm17.219 7.777c-2.641 0-4.707-.645-6.207-1.937-1.477-1.293-2.219-3.238-2.219-5.84V56.398a.67.67 0 0 1 .199-.504c.148-.133.316-.199.504-.199h3.625c.203 0 .371.066.504.199.148.129.223.297.223.504v11.43c0 1.215.281 2.133.844 2.75.582.602 1.414.898 2.5.898 1.066 0 1.891-.309 2.473-.926s.867-1.527.867-2.723v-11.43a.67.67 0 0 1 .199-.504c.148-.133.316-.199.504-.199h3.652c.203 0 .371.066.504.199s.195.297.195.504v11.457c0 2.602-.746 4.547-2.246 5.84-1.477 1.293-3.52 1.938-6.121 1.938zm12.793-.281a.73.73 0 0 1-.699-.699V56.398a.67.67 0 0 1 .195-.504c.148-.133.316-.199.504-.199h7.641c2.449 0 4.359.563 5.727 1.688 1.387 1.102 2.078 2.676 2.078 4.715 0 1.313-.309 2.426-.926 3.344s-1.469 1.617-2.555 2.105l3.848 6.906a.71.71 0 0 1 .082.309.59.59 0 0 1-.195.422.54.54 0 0 1-.395.168h-3.734c-.543 0-.926-.25-1.152-.758l-3.199-6.176h-2.863v6.234a.65.65 0 0 1-.227.504c-.129.133-.301.195-.504.195zm7.582-10.922c.863 0 1.516-.207 1.965-.617.449-.43.676-1.02.676-1.77s-.227-1.34-.676-1.77c-.43-.449-1.082-.676-1.965-.676h-3.227v4.832zm11.836 10.922a.75.75 0 0 1-.703-.699V56.398c0-.207.066-.375.195-.504a.75.75 0 0 1 .508-.199h3.762a.67.67 0 0 1 .504.199.63.63 0 0 1 .227.504v18.254c0 .203-.074.371-.227.504-.129.133-.297.195-.504.195zM24.66 58.641a3.79 3.79 0 0 1-3.789 3.793c-2.094 0-3.793-1.699-3.793-3.793a3.79 3.79 0 0 1 7.582 0zM10.707 69.496a3.79 3.79 0 0 1 3.789-3.789 3.79 3.79 0 0 1 3.789 3.789 3.79 3.79 0 0 1-3.789 3.789 3.79 3.79 0 0 1-3.789-3.789zm0 0"/><g fill-rule="evenodd"><path d="M28.727 70.797c-1.531.988-3.227 1.672-4.996 2.031a10.11 10.11 0 0 0 .559-3.332c0-.426-.023-.84-.074-1.25a10.2 10.2 0 0 0 2.176-1.062c1.645-1.062 2.945-2.578 3.75-4.363a10.14 10.14 0 0 0 .781-5.703c-.293-1.937-1.137-3.746-2.437-5.215a10.17 10.17 0 0 0-4.875-3.055 10.19 10.19 0 0 0-10.543 3.273l-.461.594c-1.937.215-3.812.766-5.547 1.613.563-1.809 1.48-3.496 2.707-4.965 1.785-2.137 4.152-3.715 6.813-4.543s5.504-.863 8.191-.113 5.094 2.258 6.938 4.348a14.46 14.46 0 0 1 3.469 7.418c.418 2.754.031 5.57-1.113 8.113a14.5 14.5 0 0 1-5.336 6.211zM7.23 56.965l3.547.438a10.08 10.08 0 0 1 .457-2.012 14.57 14.57 0 0 0-4.004 1.574zm0 0"/><path d="M6.617 57.34a14.55 14.55 0 0 1 5.031-2.039 10.17 10.17 0 0 0-.621 3.512c0 .371.02.734.055 1.094a10.46 10.46 0 0 0-2.129 1.047 10.21 10.21 0 0 0-3.754 4.367c-.805 1.785-1.074 3.766-.781 5.699s1.141 3.746 2.438 5.215 2.992 2.527 4.875 3.055 3.887.5 5.754-.082a10.16 10.16 0 0 0 4.789-3.191l.465-.598a16.9 16.9 0 0 0 5.551-1.621 14.56 14.56 0 0 1-2.711 4.977c-1.789 2.137-4.156 3.715-6.816 4.543s-5.504.867-8.187.113a14.48 14.48 0 0 1-6.941-4.344C1.785 76.996.582 74.422.164 71.664s-.031-5.57 1.113-8.113 3-4.695 5.34-6.211zM28.09 71.168a.25.25 0 0 1-.055.031.25.25 0 0 0 .055-.031zm0 0"/></g></svg>
```

Comment:
For plain and line icons, please merge the paths into one path if possible

ref: https://github.com/devicons/devicon/wiki/SVG-Standards

Reply:
done with https://github.com/devicons/devicon/pull/1457/commits/7b0f4c6cee29cdde9dfbee6c9f69c4e409405125